scrapers/USA/MO/index.js

```diff
@@ -0,0 +1,171 @@
+    'Butler County',
+    'Caldwell County',
+    'Callaway County',
+    'Camden County',
+    'Cape Girardeau County',
+    'Carroll County',
+    'Carter County',
+    'Cass County',
+    'Cedar County',
+    'Chariton County',
+    'Christian County',
+    'Clark County',
+    'Clay County',
+    'Clinton County',
+    'Cole County',
+    'Cooper County',
+    'Crawford County',
+    'Dade County',
+    'Dallas County',
+    'Daviess County',
+    'DeKalb County',
+    'Dent County',
+    'Douglas County',
+    'Dunklin County',
+    'Franklin County',
+    'Gasconade County',
+    'Gentry County',
+    'Greene County',
+    'Grundy County',
+    'Harrison County',
+    'Henry County',
+    'Hickory County',
+    'Holt County',
+    'Howard County',
+    'Howell County',
+    'Iron County',
+    'Jackson County',
+    'Jasper County',
+    'Jefferson County',
+    'Johnson County',
+    'Knox County',
+    'Laclede County',
+    'Lafayette County',
+    'Lawrence County',
+    'Lewis County',
+    'Lincoln County',
+    'Linn County',
+    'Livingston County',
+    'Macon County',
+    'Madison County',
+    'Maries County',
+    'Marion County',
+    'McDonald County',
+    'Mercer County',
+    'Miller County',
+    'Mississippi County',
+    'Moniteau County',
+    'Monroe County',
+    'Montgomery County',
+    'Morgan County',
+    'New Madrid County',
+    'Newton County',
+    'Nodaway County',
+    'Oregon County',
+    'Osage County',
+    'Ozark County',
+    'Pemiscot County',
+    'Perry County',
+    'Pettis County',
+    'Phelps County',
+    'Pike County',
+    'Platte County',
+    'Polk County',
+    'Pulaski County',
+    'Putnam County',
+    'Ralls County',
+    'Randolph County',
+    'Ray County',
+    'Reynolds County',
+    'Ripley County',
+    'St. Charles County',
+    'St. Clair County',
+    'St. Francois County',
+    'St. Louis County',
+    'Ste. Genevieve County',
+    'Saline County',
+    'Schuyler County',
+    'Scotland County',
+    'Scott County',
+    'Shannon County',
+    'Shelby County',
+    'Stoddard County',
+    'Stone County',
+    'Sullivan County',
+    'Taney County',
+    'Texas County',
+    'Vernon County',
+    'Warren County',
+    'Washington County',
+    'Wayne County',
+    'Webster County',
+    'Worth County',
+    'Wright County'
+  ],
+  async scraper() {
+    let counties = {};
+    const $ = await fetch.page(this.url);
+    const $table = $($('table')[1]);
+
+    const $trs = $table.find('tr:not(:first-child)');
+    $trs.each((index, tr) => {
+      const $tr = $(tr);
+      let countyName = parse.string($tr.find('td:nth-child(1)').text());
+      countyName = this._countyMap[countyName] || countyName;
+
+      const casesState = parse.number($tr.find('td:nth-child(2)').text()) || 0;
+      const casesOther = parse.number($tr.find('td:nth-child(3)').text()) || 0;
+      if (countyName !== 'UNASSIGNED') {
+        countyName = transform.addCounty(countyName);
+      }
```

Comment:
So for this, you previously had stuff matching on `TBD` -- we don't want to ignore that, it needs to be added into a separate object with `{ county: UNASSIGNED }` -- the constant from above (see the code comment at the top of the file).

Reply:
Thanks for the clarification.  Now I get it.